gold/

2012-11-28  H.J. Lu  <hongjiu.lu@intel.com>

	* options.h (General_options): Add -fuse-ld= for GCC linker
	option compatibility.

ld/

2012-11-28  H.J. Lu  <hongjiu.lu@intel.com>

	* lexsup.c (ld_options): Add fuse-ld= for GCC linker option
	compatibility.

diff --git a/gold/options.h b/gold/options.h
index 50762a5..674657e 100644
--- a/gold/options.h
+++ b/gold/options.h
@@ -1178,6 +1178,10 @@ class General_options
   DEFINE_special(end_lib, options::TWO_DASHES, '\0',
                  N_("End a library "), NULL);
 
+  DEFINE_string(fuse_ld, options::ONE_DASH, '\0', "",
+		N_("Ignored for GCC linker option compatibility"),
+		"");
+
   // The -z options.
 
   DEFINE_bool(combreloc, options::DASH_Z, '\0', true,
diff --git a/ld/lexsup.c b/ld/lexsup.c
index c6baebe..bb51f61 100644
--- a/ld/lexsup.c
+++ b/ld/lexsup.c
@@ -175,6 +175,9 @@ static const struct ld_option ld_options[] =
     '\0', NULL, N_("Ignored for GCC LTO option compatibility"),
     ONE_DASH },
 #endif /* ENABLE_PLUGINS */
+  { {"fuse-ld=", required_argument, NULL, OPTION_IGNORE},
+    '\0', NULL, N_("Ignored for GCC linker option compatibility"),
+    ONE_DASH },
   { {"Qy", no_argument, NULL, OPTION_IGNORE},
     '\0', NULL, N_("Ignored for SVR4 compatibility"), ONE_DASH },
   { {"emit-relocs", no_argument, NULL, 'q'},
